psl/builtin-connectors/src/planetscale.rs

```diff
@@ -1,15 +1,12 @@
 use crate::mysql_datamodel_connector;
-use psl_core::{
-    datamodel_connector::{Flavour, RelationMode},
-    js_connector::JsConnector,
-};
+use psl_core::{datamodel_connector::Flavour, js_connector::JsConnector};
 
 pub(crate) static PLANETSCALE_SERVERLESS: JsConnector = JsConnector {
     flavour: Flavour::Mysql,
     canonical_connector: &mysql_datamodel_connector::MySqlDatamodelConnector,
 
     provider_name: "@prisma/planetscale",
     name: "planetscale serverless",
-    enforced_relation_mode: Some(RelationMode::Prisma),
+    enforced_relation_mode: None,
```

Comment:
This uses the default `relationMode` for MySQL (`foreignKeys`), unless another one is specified in the Prisma schema.

Reply:
Can we remove `enforecd_relation_mode` property completely now?